torch/_dynamo/variables/torch.py

```diff
@@ -572,6 +572,7 @@ def get_state_from_generator():
         elif is_from_local(self.value):
             # rewrite non-primitive args/kwargs to be included in the on-the-fly prim function
             # and rewrite args to have only proxyable args, then insert call_function
+            # TODO: support cases where device_mesh + placements specified as kwargs
```

Comment:
mby good to assert kwargs is empty (or that it just contains the 1 bool you expect, etc)

Reply:
Actually it's not quite hard to directly support kwargs so I added that directly